src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java

```diff
@@ -52,6 +52,7 @@ public SecurityWebFilterChain securityWebFilterChain(
                                            "/**.css",
                                            "/**.png",
                                            "/health-information/fetch/**/attachments/**",
+                                           "/v1/consent-requests/on-init",
```

Comment:
It shouldn't be in `WHITELISTED_URLS` as we have authentication on this. It should rather be in `SERVICE_ONLY_URLS`, if I am not wrong.

Reply:
Please let me know if I am missing any context.